src/pages/index.tsx

```diff
@@ -160,6 +161,9 @@ const HomePage: FC<IProps> = ({
           exit={{ opacity: 0, y: 40 }}
           transition={{ ease: 'easeInOut', duration: 0.3 }}
         >
+          <SectionWithContainer>
+            <NaviBar />
+          </SectionWithContainer>
```

Comment:
Can you remove this `<SectionWithContainer>` and leave only the `<NaviBar />` because you are wrapping this content with .container using sectionWithcontainer and you are using .container inside of your navibar as well.